internal/utils/tlsutils.go

```diff
@@ -61,11 +61,11 @@ func CreateTLSConfiguration(tlsConfig *conf.TLSConfig) (t *tls.Config, err error
 		caCertPool = x509.NewCertPool()
 		caCertPool.AppendCertsFromPEM(caCert)
 	}
-
+	// #nosec G402
```

Comment:
Do you mind putting a comment here as to why this is required also please?

Reply:
added todo as well